userspace/libsinsp/test/events_plugin.ut.cpp

```diff
@@ -131,8 +131,13 @@ TEST_F(sinsp_with_test_input, event_sources)
 	ASSERT_FALSE(field_has_value(evt, "evt.asynctype"));
 
 	// metaevents have the "syscall" event source
-	evt = add_event_advance_ts(increasing_ts(), 1, PPME_CONTAINER_JSON_E, 1, "{\"value\": 1}");
-	ASSERT_EQ(evt->get_type(), PPME_CONTAINER_JSON_E);
```

Comment:
Was it intentional to use the deprecated event type `PPME_CONTAINER_JSON_E`? Changed it to `PPME_CONTAINER_JSON_2_E`